### PR TITLE
Mention upcoming bash update in release notes

### PR DESCRIPTION
**Pull request by rimrul**
While not every user reads the release notes, this should at least help to
reduce the fallout from the planned bash update.

### CODE REVIEW

File: ReleaseNotes.md
Comment (dscho) [inline ReleaseNotes.md:46]: If we move it into the "Changes since..." section, it will be part of the announcement mail...

Comment (rimrul) [reply ReleaseNotes.md:46]: That's not the worst idea I've ever heard. Give me a few minutes to force push an update.

**Review by dscho — APPROVED** [review]

File: ReleaseNotes.md
Comment (dscho) [reply ReleaseNotes.md:46]: Before hitting the merge button, one last idea: maybe mention that we're on Bash 4.4 right now?

Comment (rimrul) [reply ReleaseNotes.md:46]: That probably wouldn't hurt. I've added it.

**Review by dscho — APPROVED** [review]

**Comment by dscho** [comment]
@rimrul did you plan on working on the Bash upgrade?

**Pull request merged**
